.gitignore

```diff
@@ -22,3 +22,4 @@ settings.xml
 /target/
 /*/target/
 /shaded/*/target/
+bin
```

Comment:
This would ignore all bin folders in all subfolders (e.g. it would ignore `org.apache.calcite.encodings.bin....` folder if one is created). Do you intend that?

Reply:
Hrm, I did not. I don't think we have such folders in avatica, but that's not great either. Let me revert that.

Thanks @vlsi